### ISSUE TITLE
[BUG] I get a loop of errors after creation the documentation

### ISSUE THREAD

**Issue by GhadaAJIMI** [compodoc/compodoc#1264]
After I generate the documentation with the command: 
compodoc -p tsconfig.app.json -s

I get this error:
Copy main resources
Documentation generated in ./documentation/ in 10.203 seconds using gitbook theme
Serving documentation from ./documentation/ at http://127.0.0.1:8080

ERROR: Error: UNKNOWN: unknown error, watch
    at FSWatcher._handle.onchange (node:internal/fs/watchers:204:21) {
  errno: -4094,
  syscall: 'watch',
  code: 'UNKNOWN',
  filename: null
}

The error is repeated many times and I don't know what is the cause of it , can anyone help me plz

**Comment by vogloblinsky** [comment]
Hi,
in which version of Node.js, Compodoc ?

**Comment by GhadaAJIMI** [comment]
I found out that it is related to the number of file watchers in my pc.
When I launch angular with "ng serve", node js with "nodemon" and the tests and the documentation, there is so much files to watch and there isn't enought watchers for all of them.
I found how to add more in linux but not in windows.
As I launch my front and back, the cpu used reaches 75% and this makes the wachers loop error.
I still need help to find a solution, it will be nice to get some help
Thx
